_utils/generate-data-small.sh

```diff
@@ -1,9 +1,16 @@
 # Data generation data for groupby 0.5GB
 
-mkdir data
+mkdir -p data
 cd data/
 Rscript ../_data/groupby-datagen.R 1e7 1e2 0 0
 Rscript ../_data/join-datagen.R 1e7 0 0 0
+
+cp G1_1e7_1e2_0_0.csv G1_1e9_1e2_0_0.csv
+cp J1_1e7_1e1_0_0.csv J1_1e9_1e3_0_0.csv
+cp J1_1e7_1e4_0_0.csv J1_1e9_1e6_0_0.csv
+cp J1_1e7_1e7_0_0.csv J1_1e9_1e9_0_0.csv
+cp J1_1e7_NA_0_0.csv J1_1e9_NA_0_0.csv
```

Comment:
What is actually a purpose of that?
We could alternatively run 1e5, 1e6, 1e7 in CI rather then faking bigger size

Reply:
Mostly just to trigger solutions that have different set up logic for larger data sizes. duckdb and pydatatable (apparently) are scripts for this. 
I don't think there is a benefit to running more smaller benchmarks. The results for CI are just to check that solutions run and don't break report generation.

Reply:
I also don't want to run the actual 50GB benchmarks since then it takes forever for the regression script to finish for each solution. In addition, chances are the GitHub machines will end up running out of storage/memory